README.adoc

```diff
@@ -118,12 +79,55 @@ java -jar target/optaweb-vehicle-routing-backend-*.jar \
 [CAUTION]
 .Using large OSM files
 ====
-For best user experience it is recommended to use smaller regions like individual European or US states.
+For best user experience, use smaller regions like individual European or US states.
 Using OSM files larger than 1 GB will require significant RAM size
 and take a lot of time (up to several hours) for the initial processing.
 ====
 
-== Creating Custom Data Sets
+
+=== Run Frontend
+
+. Install `npm`. If you're on Fedora, you can do this by running:
++
+[source,shell]
+----
+sudo dnf install npm
+----
+
+. Install dependencies:
++
+[source,shell]
+----
+cd optaweb-vehicle-routing-frontend
+npm install
+----
+
+. Run frontend:
++
+[source,shell]
+----
+npm start
+----
+
+. Open http://localhost:3000/ . By default, the `npm start` command attempts to open this URL in your default browser.
```

Comment:
I will need to keep this space in. Some ADOC reading applications might otherwise include the full stop in the URL, making it incorrect.

Reply:
I disagree. I admit there might be such Asciidoc apps out there but if they include the full stop then it's a bug. It should be reported and fixed. I don't think the possibility of a buggy software justifies making this `typographic mistake .`

An acceptable workaround (avoiding a full stop immediately following a URL) would be to rearrange the sentence so that it doesn't end with the URL. For example, `Open http://localhost:3000/ in a web browser.`. What do you think?

Related discussion on stack exchange: https://ux.stackexchange.com/questions/15226/should-a-sentence-ending-with-a-url-terminate-with-a-period.

Reply:
All right, I'll rephrase.

Reply:
Pushed.

Reply:
Not pushed.

Reply:
Should be pushed now